.github/workflows/ci.yaml

```diff
@@ -0,0 +1,79 @@
+# THIS FILE WAS AUTOMATICALLY GENERATED, PLEASE DO NOT EDIT.
+#
+# Generated on 2023-09-29T16:23:25Z by kres latest.
+
+name: default
```

Comment:
@frezbo I guess this means we should disable drone and enable gh actions? Anything I need to do manually?

Reply:
yes, probablt delete version.gi if it exists and let kres re-create

Reply:
I rekres'd it just today, but I guess we need your dummy workflow to be there first, so that this PR kicks in a GH actions build?

Reply:
probably once more, since i pushed a fix sometime back